assets/src/scripts/utils.js

```diff
@@ -69,6 +69,55 @@ export function deltaDays(date1, date2) {
  */
 export function setEquality(set1, set2) {
     let sizeEqual = set1.size === set2.size;
-    let itemsEqual = [...set1].every(x => {return set2.has(x)});
+    let itemsEqual = [...set1].every(x => {
+        return set2.has(x);
+    });
     return sizeEqual && itemsEqual;
 }
+
+/**
+ * Wrap long svg text labels into multiple lines.
+ * From: https://bl.ocks.org/ericsoco/647db6ebadd4f4756cae
```

Comment:
Code courtesy bl.ocks.org, so I didn't write any tests. Hopefully that's not a bad excuse?

Reply:
It's a bad excuse 😁 .  Should probably add some and make it more ES6 friendly.

Reply:
I concur with that sentiment.